http/server.lua

```diff
@@ -685,21 +729,22 @@ local function process_client(self, s, peer)
             s:write('HTTP/1.0 100 Continue\r\n\r\n')
         end
 
-        local logreq = self.options.log_requests and log.info or log.debug
+        local route = self:match(p.method, p.path)
```

Comment:
Add comment describing what you did here!

Something like:
`-- HACK: we're in a server context, but prematurely dispatch route (router responsibility) for getting logging options which override server ones' `.

Reply:
I think this is obvious and no comment is needed.